### ISSUE TITLE
Futcher line

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#6369]
**Preferred term label:**
Futcher line

**Synonyms**
Voigt lines
Pigmentary demarcation lines

**Definition (free text, please give PubMed ID)**
A linear discontinuity in intensity of pigmentation on the upper arm and deltoid area of blacks. It is located on the lateral aspect of the arm and marks the junction between the dorsal and ventral parts of the extremity.
OMIM:137000

**Parent term (use hpo.jax.org/app)**
Hyperpigmentation of the skin HP:0000953

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:137000

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

**Comment by nicolevasilevsky** [comment]
related to https://github.com/monarch-initiative/mondo/issues/2148

**Comment by pnrobinson** [comment]
I think this is a normal finding
https://omim.org/entry/137000
I think that we can mark this OMIM entry as "trait" rather than disease or abnormal phenotypic feature. I am going to add it to the omit list.
